### PR TITLE
changed test_util.py and util.py files

### PR DESCRIPTION
**Pull request by veda10**

### CODE REVIEW

File: netcontrold/lib/util.py [hunk 1]
```diff
@@ -63,7 +63,7 @@ def cpuinfo():
         regex = re.compile('^(.*?)\s*:\s*(.*)')
         (param, val) = regex.match(line).groups()
 
-        if param == 'processor' and val == '':
+        if (param == 'processor' and val == '') or (param == 'core id' and val == '') or (param == 'physical id' and val == ''):
             raise ValueError("Value cannot be null")
```
Comment (gmuthukr) [inline netcontrold/lib/util.py]: Say which value can not be null .. "Value %s cannot be null" % param ?

File: netcontrold/lib/util.py [hunk 2]
```diff
@@ -63,7 +63,7 @@ def cpuinfo():
         regex = re.compile('^(.*?)\s*:\s*(.*)')
         (param, val) = regex.match(line).groups()
 
-        if param == 'processor' and val == '':
+        if (param == 'processor' and val == '') or (param == 'core id' and val == '') or (param == 'physical id' and val == ''):
```
Comment (gmuthukr) [inline netcontrold/lib/util.py:66]: Say which value can not be null .. "Value %s cannot be null" % param ?

**Review by gmuthukr — APPROVED** [review]
Looks good except for a minor print info and that is fine for the commit.

File: netcontrold/lib/util.py (hunk 2 above)
Comment (veda10) [reply netcontrold/lib/util.py:66]: ok

**Pull request merged**